src/main/java/io/phdata/streamliner/schemadefiner/configbuilder/ScriptCommand.java

```diff
@@ -146,6 +153,17 @@ private static void build(
               });
       writeSchemaMakeFile(
           configuration, configDiff, typeMapping, templateDirectory, outputDirectory);
+
+      try {
+        /*intentionally kept inside try catch block.
+        any exception while generating the delta-change-summary should not block the run. */
+        generateDeltaChangeSummary(configDiff, configuration, typeMapping, outputDirectory);
+      } catch (Exception e) {
+        String errorMsg = "Error generating delta-change-summary file.";
```

Comment:
> String errorMsg = "Error generating delta-change-summary file.";

Let's change this to:

> String errorMsg = "Error generating delta-change-summary file: " + e;

which will give us the full exception name and error message in the file.

Reply:
updated